packages/cli/src/base.ts

```diff
@@ -228,10 +235,11 @@ export abstract class BaseCommand extends Command {
   async finally(arg: Error | undefined): Promise<any> {
     try {
       if (arg) {
-        console.error('received error while cleaning up', arg)
```

Comment:
since error logs are shown to users it gets noise if we logg this. so set to debug instead

this was recently added in last release